tests/test_figure_resampler_selenium.py

```diff
@@ -71,6 +71,7 @@ def test_multiple_tz(driver, multiple_tz_figure):
         raise e
     finally:
         proc.terminate()
+    multiple_tz_figure.stop_server()
```

Comment:
Why not in the finally clause (gets always executed)

Reply:
good catch!

Reply:
Will fix this myself